src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import java.util.HashMap;
+import java.util.Map;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Cart {
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private long id;
+
+    @Column(name = "products")
+    private Map<Product, Integer> products = new HashMap<>();
```

Comment:
Może być ciężko odwzorować mapę jako pojedyńczą kulumnę. Możesz zerknąć na:
https://www.baeldung.com/hibernate-persisting-maps
lub  zostawić implementacje klasy Cart pustą dla kogoś kto weźmie zadanie z implementacją tej klasy.

Reply:
Cos tu się dzieje?

Reply:
Nie widzę, żeby to było zmienione, a Ty @karoldd9 się też nie odniosłeś do tego komentarza i ponownie przypisałeś do review do mnie.